### DISCUSSION TITLE
Libreddit instance

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#1043] [Ideas]
Consider add this:
https://github.com/spikecodes/libreddit#:~:text=An%20alternative%20private%20front-end,r%2Funpopularopinion%20without%20being%20tracked.

Alternative to reddit more private 

Also a question:
Is IFR prevent reddit to track user or visitor?

**Comment by mxvin** [comment]
Both project have a different approach. libreddit is like 'proxy' plus its frontend and infinity is only the frontend and rely on reddit.com. I think your request is irrelevant for this project.

**Comment by ghost** [comment]
> Both project have a different approach. libreddit is like 'proxy' plus its frontend and infinity is only the frontend and rely on reddit.com. I think your request is irrelevant for this project.

what i mean is just simply make sure that Reddit won't track us that is one of purpose of  IFR right? @mxvin

**Comment by flexagoon** [comment]
@NSTAdventure Infinity is a client for which you need an account, since it allows you to post things. Libreddit doesn't support accounts, and all of your subscriptions are stored locally. If you want a private Reddit client which you can use without an account, try [Stealth](https://f-droid.org/packages/com.cosmos.unreddit/) (although keep in mind that you will not be able to post or comment)

**Reply by ghost** [reply]
Thanks i also use stealth
I hate those reddit crap and it's controversy i don't want get involved with such
